gdnative-core/src/core_types/geom/transform.rs

```diff
@@ -304,6 +304,14 @@ impl Mul<Transform> for Transform {
     }
 }
 
+impl MulAssign<Transform> for Transform {
+    #[inline]
+    fn mul_assign(&mut self, rhs: Self) {
+        self.origin = self.xform(rhs.origin);
+        self.basis = self.basis * rhs.basis;
+    }
+}
```

Comment:
With this new `*=` operator, we could implement `*` in terms of it:
```rs
impl Mul<Transform> for Transform {
    type Output = Transform;
    #[inline]
    fn mul(self, rhs: Self) -> Self::Output {
        self *= rhs;
        self
    }
}
```

Reply:
Good point. The receiver needs to be changed to `mut self` (no ampersand) though I suppose?

Reply:
Yes, you're right.